src/test/resources/templates/email/feedbackToInterviewer.txt

```diff
@@ -0,0 +1,19 @@
+Thank you for practicing your interviewing skills using InterviewMe on {{formatted_date}}. Your interviewee has submitted feedback for your interview, which you can view below. 
```

Comment:
Why are these email templates here?

Reply:
These are used for the tests

Reply:
Seems like a lot of duplication, but l guess the contents of the test templates don't matter.